### PR TITLE
Actifile.ActifileAgent version 2.0.65.0

### PR DESCRIPTION
**Pull request by DanPrs**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/15798)

### CODE REVIEW

**Comment by KaranKad** [comment]
You can only edit one manifest per pr.

**Comment by DanPrs** [comment]
> You can only edit/add one manifest per pr.

Sorry about that. Thanks!

File: manifests/a/Actifile/ActifileAgent/2.0.65.0/Actifile.ActifileAgent.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate
+
+PackageIdentifier: Actifile.ActifileAgent
+PackageVersion: 2.0.65.0
+Installers:
+- InstallerLocale: en-US
+  Architecture: neutral
+  InstallerType: msi
+  InstallerUrl: http://app.actifile.com/home/downloadagentmsi
```
Comment (ItzLevvie) [inline manifests/a/Actifile/ActifileAgent/2.0.65.0/Actifile.ActifileAgent.installer.yaml:9]:
```suggestion
  InstallerUrl: https://app.actifile.com/home/downloadagentmsi
```

File: manifests/a/Actifile/ActifileAgent/2.0.65.0/Actifile.ActifileAgent.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate
+
+PackageIdentifier: Actifile.ActifileAgent
+PackageVersion: 2.0.65.0
+Installers:
+- InstallerLocale: en-US
+  Architecture: neutral
```
Comment (ItzLevvie) [inline manifests/a/Actifile/ActifileAgent/2.0.65.0/Actifile.ActifileAgent.installer.yaml:7]:
```suggestion
  Architecture: x86
```

File: manifests/y/Yubico/Authenticator/5.0.5/Yubico.Authenticator.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+Installers:
+- Architecture: x64
+  InstallerUrl: https://developers.yubico.com/yubioath-desktop/Releases/yubioath-desktop-5.0.5-win64.msi
+  InstallerSha256: 580C825A5AF14989056144B90706E01070BF09F81EAABD0BE10BCEE2508EB2B9
+  ProductCode: '{C4938DCC-540E-43DB-BDC2-962A94308617}'
+  InstallerType: msi
+  InstallerSwitches:
+    Silent: /qn /norestart
+    SilentWithProgress: /qn /norestart
+- Architecture: x32
```
Comment (ItzLevvie) [inline manifests/y/Yubico/Authenticator/5.0.5/Yubico.Authenticator.installer.yaml:23]:
```suggestion
- Architecture: x86
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/y/Yubico/Authenticator/5.0.5/Yubico.Authenticator.installer.yaml (hunk 3 above)
Comment (DanPrs) [reply manifests/y/Yubico/Authenticator/5.0.5/Yubico.Authenticator.installer.yaml]: Thank you!

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**